### ISSUE TITLE
Monitor GPO Changes, Builtin to get GPO Displayname

### ISSUE THREAD

**Issue by DonDom86** [EvotecIT/PSWinReporting#83]
Examples/RunMe-FindEventsGroupPolicyChangesWithConfig.ps1

Hi saw you example.
Is there an builtin way to convert IDs to the DisplayName of a GPO?

Thanks

**Comment by PrzemyslawKlys** [comment]
No, Events don't store it from what I remember. But I will most likely add functionality that queries it live in PSEventViewer

**Comment by DonDom86** [comment]
No the event do not store them directly
